### PR TITLE
{tools}[foss/2023a] Faiss v1.7.4, t-SNE-CUDA v3.0.1 w/ CUDA 12.1.1

### PR DESCRIPTION
**Pull request by jfgrimm**
requires:

- [x] https://github.com/easybuilders/easybuild-easyconfigs/pull/19666

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
gpu07.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, 2 x NVIDIA NVIDIA A40, 535.86.10, Python 3.6.8
See https://gist.github.com/jfgrimm/c5915572b24464f5dba049712511e15e for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
gpu07.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, 2 x NVIDIA NVIDIA A40, 535.86.10, Python 3.6.8
See https://gist.github.com/jfgrimm/b344a61fe513fe9ba7cd4f4968aaa484 for a full test report.

File: easybuild/easyconfigs/t/t-SNE-CUDA/t-SNE-CUDA-3.0.1-foss-2023a-CUDA-12.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    '-DBUILD_PYTHON=ON',
+    '-DWITH_ZMQ=FALSE',
+    '-DWITH_MKL=OFF',
+    '-DCMAKE_CUDA_ARCHITECTURES="%(cuda_cc_cmake)s"',
+]
+
+configopts = ' '.join(_copts)
+
+install_cmd =  ('cd %(builddir)s/easybuild_obj/python &&'
+                ' python -m pip install --prefix=%(installdir)s --no-build-isolation .')
```
Comment (jfgrimm) [inline easybuild/easyconfigs/t/t-SNE-CUDA/t-SNE-CUDA-3.0.1-foss-2023a-CUDA-12.1.1.eb:49]:
```suggestion
install_cmd = ('cd %(builddir)s/easybuild_obj/python &&'
               ' python -m pip install --prefix=%(installdir)s --no-build-isolation .')
```

**Comment by emdrago** [comment]
Test report by @emdrago
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
gput052 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz, 1 x NVIDIA NVIDIA GeForce RTX 2080 Ti, 535.54.03, Python 3.9.13
See https://gist.github.com/emdrago/3729452d9411286fe8d15bd627eae272 for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**